lib/lit/i18n_backend.rb

```diff
@@ -83,8 +85,10 @@ def lookup(locale, key, scope = [], options = {})
 
     def store_item(locale, data, scope = [])
       if data.respond_to?(:to_hash)
-        data.to_hash.each do |key, value|
-          store_item(locale, value, scope + [key])
+        ActiveRecord::Base.transaction do
```

Comment:
Isn't it redundant? Transactions at `store_translations` and `load_translations_to_cache` are not enough?